app/policies/evaluation_policy.rb

```diff
@@ -1,28 +1,44 @@
 class EvaluationPolicy < ApplicationPolicy
   def show?
-    (leader_of_the_group? || leader_of_the_resort?) && !off_season?
+    (leader_of_the_group? || leader_of_the_resort? || pek_admin?) && !off_season?
   end
 
   alias current? show?
   alias table? show?
 
   def edit?
-    leader_of_the_group? && application_season?
+    leader_of_the_group? && !off_season?
```

Comment:
I think we don't need to allow to edit this in `evaulation_season` for the leader of the group. 
I'm a bit confused, so please correct me if I'm wrong.
We have 3 season, right?
1. off_season - noone able to edit and create evaluation
2. application_season - the leaders create the evaluation and fill the related fields
3. evaulation_season - the RVT check the evaluation and the leader can edit if the RVT requests changes